extensions/amp-story/amp-story.md

```diff
@@ -785,6 +785,62 @@ You can apply multiple entrance animations on one element (for example, an eleme
 If a composed animation is supposed to start after the end of a separate element's animation, make sure that all nested elements that compose the animation have the attribute `animate-in-after` set to the same `id`.
 {% endcall %}
 
+## Branching
+
+Stories support fragment paramenters in the url. By utilizing this, you can create stories that start from somewhere other than the beginning, share specific pages of stories, or even allow users to jump around within a story. An example is a table of contents.
+
+### Manipulating Navigation
+
+The feature allows you to manipulate navigation within a story. When using it, navigation throughout the story from a 'branch-off point' will follow the path that a user takes. If a user navigates from `page-1` to `page-5` and then `page-6`, navigating backwards will follow this exact path (`page-6` to `page-5` and finally `page-1`). The other pages will not be exposed to the user.
+
+### Sharing Options
+
+Specific pages within a story are shareable through URLs.
+
+### URL Fragment Parameter
+
+With branching, AMP Stories now supports URLs in the form of:
+
+```
+https://www.mydomain.com/good-story/#page=<page-id>
+```
+
+where `page-id` refers to the unique id of an `amp-story-page`. You can also use the fragment parameter and the `page-id` value like an anchor link in some use cases. See [Integration with Sidebar for Stories](#integration-with-sidebar-for-stories) for an example.
+
+### Sharing a Specific Story Page
```

Comment:
This part has been removed from the code

Reply:
Ditto last comment, not sure if I'm missing something.